core/src/main/java/tc/oc/pgm/api/setting/SettingValue.java

```diff
@@ -19,19 +19,22 @@
   CHAT_ADMIN("chat", "admin"), // Send to all server operators
 
   DEATH_OWN("death", "own"), // Only send death messages involving self
+  DEATH_FRIENDS("death", "friends"), // Only send death messages involving yourself or friends
```

Comment:
You need to add all of these new settings at the end to prevent messing up existing settings.

Reply:
Oh shoot that's odd, thought I fixed all the settings. Will do 👍